.idea/codeStyles/Project.xml

```diff
@@ -48,6 +48,31 @@
       <option name="CONTINUATION_INDENT_IN_PARAMETER_LISTS" value="false" />
       <option name="WRAP_EXPRESSION_BODY_FUNCTIONS" value="1" />
     </JetCodeStyleSettings>
+    <Objective-C-extensions>
```

Comment:
This one was actually up to date in WPAndroid - it should be removed from `style-config-android`. See https://github.com/wordpress-mobile/WordPress-Android/pull/9672 for background.

Reply:
👍 missed that